language/move-bytecode-verifier/src/verifier.rs

```diff
@@ -20,6 +20,7 @@ use move_binary_format::{
 pub struct VerifierConfig {
     pub max_loop_depth: Option<usize>,
     pub treat_friend_as_private: bool,
+    pub strict_runtime_verification: bool,
```

Comment:
As discussed on the other PR lets not call this strict... There is really an association with 'strict' vs 'non-strict' which is a misfit here.

The name suggested by Sam, `paranoid_checks`, is much better and reflects what this feature is really about in the overall Move architecture.